scipy/stats/_qmc.py

```diff
@@ -985,6 +985,7 @@ def __init__(
         optimization: Optional[Literal["random-cd", "lloyd"]] = None,
         seed: SeedType = None
     ) -> None:
+        self._init = {'d': d, 'scramble': True, 'optimization': optimization}
```

Comment:
Maybe we should have another name because it's not exactly corresponding to the input as `scramble` is fixed to True. Something like `_init_integral`. This way we also know it's just for that.

I am not a fan of using that, but don't see a cleaner way for now. We will also need to add some tests (can be done later) for that at some point otherwise we might forget something as we add parameters.